packages/scripts/src/generate-readme/generator.ts

```diff
@@ -152,6 +153,27 @@ export class ReadmeGenerator {
       //Note, not populating description, doesn't exist in framework adapters
       this.defaultEndpoint = adapter.defaultEndpoint ?? ''
     }
+
+    //TODO: Does the below still belong here? Came up in source branch when rebasing on develop
+    if (this.verbose) console.log(`${this.adapterPath}: Importing src/config/index.ts`)
+    this.defaultBaseUrl = configFile.DEFAULT_BASE_URL || configFile.DEFAULT_WS_API_ENDPOINT
+
+    if (fs.existsSync(this.adapterPath + 'src/endpoint/index.ts')) {
+      if (this.verbose) console.log(`${this.adapterPath}: Importing src/endpoint/index.ts`)
+      const endpointPath = checkFilePaths([this.adapterPath + 'src/endpoint/index.ts'])
+      this.endpointDetails = await require(path.join(process.cwd(), endpointPath))
+    } else {
+      if (this.verbose) console.log(`${this.adapterPath}: Could not find enpoint details`)
+      this.endpointDetails = {}
+    }
+    // Map V3 fields to their V2 equivalents
+    if (this.frameworkVersion === 'v3') {
+      console.log(`${this.name} is a v3 adapter, converting it to v2 format for readme generation`)
+      Object.keys(this.endpointDetails).forEach((endpointName) => {
+        const endpoint = this.endpointDetails[endpointName]
+        endpoint.supportedEndpoints = [endpointName, ...(endpoint.aliases || [])]
+      })
+    }
```

Comment:
Yeah the only comment I have is to confirm whether these lines are needed. You can probably just test this by running `generate` against a few v2 and v3 adapters and confirming that the READMEs are equivalent with and without these lines.

Reply:
They were not needed, indeed only a remnant of rebasing. Removed in latest PR